jax/nn/functions.py

```diff
@@ -182,9 +182,7 @@ def gelu(x):
   <https://arxiv.org/abs/1606.08415>`_, section 2.
   """
   sqrt_2_over_pi = np.sqrt(2 / np.pi).astype(x.dtype)
-  # Does not use the power operator here.
-  # See https://github.com/google/jax/pull/3036
-  x_cubed = x * x * x
+  x_cubed = lax.integer_pow(x, 3)
```

Comment:
Optional: could probably use infix `**` now:
```suggestion
  x_cubed = x ** 3
```